src/jmh/java/dev/blaauwendraad/masker/json/JsonMaskerBenchmark.java

```diff
@@ -1,27 +1,12 @@
 package dev.blaauwendraad.masker.json;
 
 import dev.blaauwendraad.masker.json.config.JsonMaskingConfig;
-import org.openjdk.jmh.annotations.Benchmark;
-import org.openjdk.jmh.annotations.BenchmarkMode;
-import org.openjdk.jmh.annotations.Fork;
-import org.openjdk.jmh.annotations.Measurement;
-import org.openjdk.jmh.annotations.Mode;
-import org.openjdk.jmh.annotations.OutputTimeUnit;
-import org.openjdk.jmh.annotations.Param;
-import org.openjdk.jmh.annotations.Scope;
-import org.openjdk.jmh.annotations.Setup;
-import org.openjdk.jmh.annotations.Warmup;
+import org.openjdk.jmh.annotations.*;
```

Comment:
I am curious about your opinion on wildcard imports.

Reply:
My IDEA did it again :( I always configure it to not do that, but since I use a couple of different workstations, some of them still collapse those.

Reply:
Should be fixed now.